include/utilities/mdframe/handler_csv.hpp

```diff
@@ -0,0 +1,102 @@
+#ifndef NGEN_MDFRAME_HANDLERS_CSV_HPP
+#define NGEN_MDFRAME_HANDLERS_CSV_HPP
+
+#include <fstream>
+
+#include "mdframe.hpp"
+#include "cartesian.hpp"
+
+namespace ngen {
+
+inline void mdframe::to_csv(const std::string& path, bool header) const
```

Comment:
The implementations of this and `to_netcdf` should be in .cpp files under `src/`, not in headers. Among other things, that would push includes of `cartesian.hpp` and `<netcdf>` out of headers being included by other code.